_data/sw_clients.yml

```diff
@@ -71,6 +80,11 @@
           - server-time
           - monitor
           - userhost-in-names
+        SASL:
+          - dh-aes
+          - dh-blowfish
```

Comment:
Wrong again `< TingPing> Mikaela, it still does though it was planned to be removed`